src/app/loadout-builder/types.ts

```diff
@@ -72,6 +72,7 @@ export const ModPickerCategories = { ...Armor2ModPlugCategories, seasonal: 'seas
 export type ModPickerCategory = typeof ModPickerCategories[keyof typeof ModPickerCategories];
 
 export interface LockedArmor2Mod {
+  key: string;
```

Comment:
It'd be nice to document what this is.

Reply:
Done, it's basically an id of sorts for removing items and using as a jsx key. Put a comment on it, let me know if you want to rename it too. I didn't want to use id and couldn't come up with anything I liked better.